src/components/Tag.js

```diff
@@ -60,7 +61,7 @@ const Title = styled(TextElement).attrs(({ color, theme: { colors } }) => ({
   margin-bottom: 1;
 `;
 
-const Tag = ({ color, disableMenu, slug, text, title, ...props }) => {
+const Tag = ({ color, disableMenu, slug, text, title, maxValue, ...props }) => {
```

Comment:
should the `Tag` component remain generic (not worry itself about a specific usage property such as `maxValue`?)

Instead should the `maxValue` logic be handled in `UniqueTokenAttributes` and passed along with the `text` value?